src/exercises/Exercises.js

```diff
@@ -34,10 +32,9 @@ let BOOKMARKS_FOR_EXERCISE = [
 export default function Exercises({
   api,
   articleID,
-  source,
-  openArticle,
-  reloadExercises,
-  openReview,
+  backToReadingAction,
+  keepExercisingAction,
+  goBackAction,
```

Comment:
Uff. What's this "go back" action? 

ah, in is used when the user opened the "words for article" but
there are no words for that article. then he's invited to go back 
to the article and do some reading... so it should be named something like:

    `backToArticleAction`

And then there's a very subtle difference between backToReading which 
in my case brings the reader to all the articles and backToArticle action.

To simplify this I propose then that we only have two actions:
- keepExercising
- backToArticle 

And we always say: "Back to article" and allow the user to 
return to the article in case she wants to. 

Do you see what I mean?

Reply:
An alternative could be that we use a Modal for the buttons that go "back". 
And then we simply close the modal.

Reply:
I'm not quite sure. We need the three different kinds of buttons in the extension - they do three different things. We need a button for "keepExercising", we need a button for "backToReadingArticle" and in our case the "goBackAction" takes the user one step back - into the "review vocabulary" component. So, I don't think it will work for us to cut it down to two buttons. 

I'm also not quite sure i understand the modal solution. We already display everything in a modal in the extension, and I think we had some difficulties using new modals inside of the current modal - in regard to the design.

Reply:
Ok. Now I think I understand better how we can get rid of the backToReviewVocabulary. 
That one happens because there were not sufficient words to practice on the previous page.
But on that page we already know that there are insufficient words to practice, so we should not let the user go forward in the first place. Then, we don't have to go back either.

Reply:
Hi @mircealungu

I guess we can just merge the last three pull requests now, right? Or should I re-request a review from you?

Reply:
I think we can merge now, indeed!